README.md

```diff
@@ -13,6 +13,22 @@ Single-crate workspaces can use it as:
   run: # your `cargo publish` code here
 ```
 
+# Input options
+
+Every argument is optional.
+
+| Input              | Description                                                                                                                       | Default |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|
+| crate-name         | The crate whose API to check for semver                                                                                           |         |
+| crate-target       | By default, check the library target of the crate. To check a different target (e.g. a binary target), set this to `--bin <NAME>` | `--lib` |
+| version-tag-prefix | The prefix to use for the git tag for a version; the default "v" creates tags like "v1.0.0"                                       | `v`       |
+
+# Scenarios
+
+- [Use with a different version tag format](#use-with-a-different-version-tag-format)
+- [Use in workspaces with more than one crate](#use-in-workspaces-with-more-than-one-crate)
+- [Use with binary crates or crates with more than one target](#use-with-binary-crates-or-crates-with-more-than-one-target)
```

Comment:
This entire scenario shouldn't be needed -- it's an artifact of the action using its own logic to generate rustdoc JSON rather than the cargo-semver-checks' internal logic (which didn't exist at the time this part of the action was written).

Let's mark it as deprecated.
```suggestion
- [(Deprecated) Use with binary crates or crates with more than one target](#use-with-binary-crates-or-crates-with-more-than-one-target)
```